Test suite NBodyProblem-test: RUNNING...
python: can't open file 'test.py': [Errno 2] No such file or directory
*** Failed! Exception: 'file_test.txt: openFile: does not exist (No such file or directory)' (after 1 test):
0.0
[32mACCEPTED![37m
Test suite NBodyProblem-test: PASS
Test suite logged to: /home/mar/NbodyProblem_1/dist-newstyle/build/x86_64-linux/ghc-9.2.8/NBodyProblem-0.1.0.0/t/NBodyProblem-test/test/NBodyProblem-0.1.0.0-NBodyProblem-test.log
